pkg/kube/client.go

```diff
@@ -157,6 +157,17 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
+	var objPods = make(map[string][]api.Pod)
+
+	for _, value := range infos {
```

Comment:
This block should be moved into the `perform` function callback below (as in [this example](https://github.com/yan234280533/helm/compare/patch-pod...thomastaylor312:patch-pod#diff-a483d6c0863082c3df21f4aad513afe2R172)) to avoid having to update each object in a release twice